frontend/app/mixins/get_logged_in_user.js

```diff
@@ -0,0 +1,35 @@
+import Ember from "ember";
+import ajax from "frontend/helpers/ajax";
+import constants from "frontend/utils/constants";
+let mixin = Ember.Mixin.create({
+  session: Ember.inject.service(),
+  getLoggedInUser:function(){
+    let self = this;
+    let hash= {
+      success:function(data){
```

Comment:
I think the non-transition part in here should not be in a mixin but in the `session` service